.github/workflows/ttnn-post-commit.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         test-group: [
           {name: ttnn group 1, cmd: pytest $TT_METAL_HOME/tests/ttnn/unit_tests -v --splits 2 --group 1},
           {name: ttnn group 2, cmd: pytest $TT_METAL_HOME/tests/ttnn/unit_tests -v --splits 2 --group 2},
-
+          {name: ttnn stable_diffusion e2e, cmd: pytest $TT_METAL_HOME/tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model.py -k 512 -v},
```

Comment:
how many integration tests are there? Are we running the `integration_tests` folder anywhere else?

Reply:
I'm thinking a better solution could be to add a symlink to this test in the `unit_tests` directory to avoid having to update the .yaml file

Reply:
It just runs 1 test. The nightly fast/slow-dispatch pipelines run  the integration_tests as well, but I'd like to add the SD e2e test as a quick sanity-check.

Reply:
I see, that makes sense - I'll try the approach